packages/palette/src/elements/Modal/ModalBase.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useRef } from "react"
+import { createPortal } from "react-dom"
+import { FocusOn } from "react-focus-on"
+import styled, { css } from "styled-components"
+import { Flex, FlexProps } from "../Flex"
+
+const ModalWrapper = styled(Flex)<{ overlay?: boolean }>`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  /* TODO: Use a token? Or some other alternative to wildstyle z-index? */
+  z-index: 9999;
```

Comment:
Do we have a centralized z-map anywhere? I tend to think of this as application specific and it's sorta weird to put in a UI library IMO. Though it can be annoying having to provide them.

Reply:
I don't think we do, but if we did I definitely think it should go in the theme file. You can go ahead and move it there if you want.

Reply:
We're def over 900, because of old force code.